action/concrete_factory_options.go

```diff
@@ -55,5 +57,30 @@ func (c SoftLayerConfig) Validate() error {
 		return bosherr.Error("Must provide non-empty ApiKey")
 	}
 
+	err := os.Setenv("SL_API_WAIT_TIME", strconv.Itoa(c.FeatureOptions.ApiWaitTime))
```

Comment:
We set these parameters as feature options, and pass them to `softlayer-go` through environment variables, we can't directly set them as  environment variables since BOSH cleans up environment variables when invoking CPI. But CPI can set environment variables so that  `softlayer-go` can retrieve them from environment.